cv32e40p/README.md

```diff
@@ -14,3 +14,23 @@ There are README files in each directory with additional information.
 
 ## Getting Started
 Check out the Quick Start Guide in the [CORE-V-VERIF Verification Strategy](https://core-v-docs-verif-strat.readthedocs.io/en/latest/).
+
```

Comment:
This is redundant.  The proper place to put this is [$(CORE_V_VERIF)/mk/README.md](https://github.com/openhwgroup/core-v-verif/tree/cv32e40p/release/mk#running-the-environment-with-synopsys-vcs-vcs-vcs).  Look for "Running the environment with Synopsys VCS".

Reply:
Please ignore this commit.